Platform-Linux/Hardening/Install-Update/0-Bash/install-ansible-control.sh

```diff
@@ -0,0 +1,3 @@
+sudo apt-add-repository ppa:ansible/ansible
```

Comment:
With these kind of scripts I prefer if they have a EUID check for sudo and exit early rather than including `sudo xyz` this is just my preference.